### ISSUE TITLE
使用bert fine-tune, acc 93%

### ISSUE THREAD

**Issue by cuiwow** [fateleak/toutiao-text-classfication-dataset#4]
没有看到有分类代码，自己写了一个。
简单的做了数据均衡和数据增强，然后finetune了bert-chinese，划分出来的测试集上acc 93%。
目前看标签确实有一些问题，比如“股票”和“财经”标签下的数据是混杂的，最好能有对于数据标签准确率等方面的评估指标。

**Comment by liujunchangqq** [comment]
> 没有看到有分类代码，自己写了一个。
> 简单的做了数据均衡和数据增强，然后finetune了bert-chinese，划分出来的测试集上acc 93%。
> 目前看标签确实有一些问题，比如“股票”和“财经”标签下的数据是混杂的，最好能有对于数据标签准确率等方面的评估指标。

我用的huggingface的bert-base-chinese但是accuracy很低，一直不收敛。用的adam优化器，batchSize是6，请问你的参数都是哪些呢？

**Comment by imlegend1015** [comment]
我也是，用albert 微调也是不收敛，acc低到吓人

**Comment by yongyixiao** [comment]
你们是将标题和关键词合起来做训练数据吗

**Comment by imlegend1015** [comment]
我是合起来坐的训练

**Comment by yongyixiao** [comment]
哦哦，好的，谢谢哈




------------------&nbsp;原始邮件&nbsp;------------------
发件人: "imlegend1015"<notifications@github.com&gt;; 
发送时间: 2020年6月18日(星期四) 下午5:54
收件人: "skdjfla/toutiao-text-classfication-dataset"<toutiao-text-classfication-dataset@noreply.github.com&gt;; 
抄送: "Ｍvp_FerryMan"<810700982@qq.com&gt;; "Comment"<comment@noreply.github.com&gt;; 
主题: Re: [skdjfla/toutiao-text-classfication-dataset] 使用bert fine-tune, acc 93% (#4)





 
我是合起来坐的训练

**Comment by yongyixiao** [comment]
那这个关键词是不是就不需要用分词，直接把各个关键词分别当成一个独立的词来去算向量吗

**Comment by mattNest** [comment]
What's the hyperparemeters setting for acc to go to 93%? I follow the settings from the BERT doc but can't get acc 90% above..

**Comment by findoctor** [comment]
can you share your repo or maybe the hyper-param settings ? I tried BERT+FC and BERT+CNN, both ended up about 84% on the test set.

**Comment by mattNest** [comment]
Actually I found out that using the hyperparameters provided by the doc can get 93% acc.

**Comment by findoctor** [comment]
> Actually I found out that using the hyperparameters provided by the doc can get 93% acc.

I didn't find such doc. Can you share with me the link ? thx
